docs/about/history.rst

```diff
@@ -25,6 +25,8 @@ Changed
 .. bullet_list::
 
    -  Added new CPU-based collision provider - *Baked FFT Data*.
+   -  Add support for rendering in scene view to *Underwater Renderer*.
```

Comment:
Not sure if also worth saying "edit mode". Up to you

Reply:
Sure. Done.